Fix: scroll to top

millasml opened the pull request:
make scroll to top appear again

sheehui reviewed (approved):
yayy but how did this fix haha

millasml commented:
> yayy but how did this fix haha

it had to do with the ref i was passing in - when i added the check for `entries.length === 0`, it made it such that the ref was not initialized to the div that I wanted. I fixed it by moving that div out of that check, so the div always renders no matter what `entries.length` is!
